test_unstructured/partition/pdf_image/test_pdf.py

```diff
@@ -1545,3 +1545,45 @@ def test_document_to_element_list_sets_category_depth_titles():
     assert elements[1].metadata.category_depth == 2
     assert elements[2].metadata.category_depth is None
     assert elements[3].metadata.category_depth == 0
+
+
+@pytest.mark.parametrize("file_mode", ["filename", "rb", "spool"])
+@pytest.mark.parametrize(
+    ("strategy", "origin"),
+    # fast: can't capture the "intentionally left blank page" page
+    # others: will ignore the actual blank page
+    [
+        (PartitionStrategy.FAST, {"pdfminer"}),
+        (PartitionStrategy.FAST, {"pdfminer"}),
+        (PartitionStrategy.HI_RES, {"yolox", "pdfminer", "ocr_tesseract"}),
+        (PartitionStrategy.OCR_ONLY, {"ocr_tesseract"}),
+    ],
+)
+def test_partition_pdf_with_password(
```

Comment:
Please clean up this test. The `origin` param is unused, `(PartitionStrategy.FAST, {"pdfminer"})` is a repeated parameter, and add type hints. I made fixes to this test previously [here](https://github.com/Unstructured-IO/unstructured/pull/3876/files)